src/axis.ts

```diff
@@ -0,0 +1,48 @@
+import * as d3 from "d3";
+import { Pareto} from "./pareto";
+
+ export async function drawLine(pareto: Pareto) {
+    d3.selectAll("path").remove();
+    d3.selectAll("g").remove();
+
+    const maxValueHeight:number = pareto.maxValue;
+    const maxValueWidth:any = [];
+    pareto.stackedBars.forEach((p) => {
+        maxValueWidth.push(p.label);
+    });
+  
+    var svg = d3.select("svg");
+    const valueAxisHeight:any = document.querySelector("#svg");
+    const valuePixels:any = valueAxisHeight.getBoundingClientRect();
```

Comment:
I would name it as svgBoundingClientRect

Reply:
it returns pixels I think this is self-explanatory